plugin/trino-cassandra/src/main/java/io/trino/plugin/cassandra/CassandraType.java

```diff
@@ -682,4 +683,15 @@ public int hashCode()
     {
         return Objects.hash(kind, trinoType, argumentTypes);
     }
+
+    @Override
+    public String toString()
+    {
+        String string = format("%s(%s", kind, trinoType);
```

Comment:
there is no performance benefit to doing so, as java internally replaces this with StringBuilder, and it doesn't improve readability IMO.

Reply:
My original comment is on the `String string` naming - `String toStirng` is better.

If we're arguing about performance benefit, why use String.format here instead of concatenation?
```
//String string = format("%s(%s", kind, trinoType);
String toString = kind + "(" _ trinoTyper;
```
Ultimately, my comment is: this `toString()` method differs from all others in style + variable naming - arguing about the performance isn't really my point - it's consistency. Thanks for pointing out Java (sometimes) optimizes `"a" +  "b" + "c"` to use StringBuilder - I suggested StringBuilder to match the existing style, and as a (possible) workaround for the `String string` naming.

I'd still suggest you rename that `String toString`, whatever else about String.format + builder, etc 👍

Reply:
Annnnd now I see it's merged :-)
Ignore me! ;-)